### ISSUE TITLE
Simulating Conditional Jumps (listings 48, 49, 50) 

### ISSUE THREAD

**Issue by hintron** [hintron/computer-enhance#7]
[Watch the next lesson][1] about simulating conditional jumps and implement listings 48, 49, and 50.

The biggest change will be keeping track of the instruction pointer. This might require re-architecting how I loop through instructions, or perhaps I will need to build some kind of lookup table for addresses and instructions.

[1]: https://www.computerenhance.com/p/simulating-conditional-jumps

**Comment by hintron** [comment]
Listing 48 was implemented in commits bbaedb0 and e70e3af.

**Comment by hintron** [comment]
Listing 49 was implemented with commits c9c53d0...b79360e.

**Comment by hintron** [comment]
Listing 50 was implemented with commits f13d8e7 and 271ad51.

NOTE: I had an issue because I was erroneously telling the loop instructions to set flags when modifying CX, because I was confused and thought that the zero flag check was for that CX decrement and not for the instruction before. Once I stopped modifying the flags when updating CX, things worked.

Also, I implemented loop and loopz/loope, just because I could, but they are untested. It's also in a state to easily implement the rest of the jump instructions.
